### ISSUE TITLE
Failure downloading http://supergsego.com/apache/tomcat/tomcat-8/v8.5.9/bin/apache-tomcat-8.5.9.tar.gz

### ISSUE THREAD

**Issue by kevinpeterson** [smart-on-fhir/installer#4]
I'm getting this:

    Failure downloading http://supergsego.com/apache/tomcat/tomcat-8/v8.5.9/bin/apache-tomcat-8.5.9.tar.gz

When I run:

    sudo ansible-playbook  -c local -i 'localhost,' -vvvv site.yml

I checked that URL and got a 404.

**Comment by tmccrimm** [comment]
I just got the same thing. Maybe the server is down.

On Feb 8, 2017 4:11 PM, "Kevin Peterson" <notifications@github.com> wrote:

I'm getting this:

Failure downloading
http://supergsego.com/apache/tomcat/tomcat-8/v8.5.9/bin/apache-tomcat-8.5.9.tar.gz

When I run:

sudo ansible-playbook  -c local -i 'localhost,' -vvvv site.yml

I checked that URL and got a 404.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/smart-on-fhir/installer/issues/4>, or mute the thread
